<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QPushButton" name="view_btn">
       <property name="text">
        <string>View all</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="edit_btn">
       <property name="text">
        <string>Edit selected</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="delete_selected">
       <property name="text">
        <string>Delete selected</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Library</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QListWidget" name="view_list"/>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="clear_view_btn">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="sort_btn">
       <property name="text">
        <string>Sort by year</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="add_btn">
       <property name="text">
        <string>Add new</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="choose_btn">
       <property name="text">
        <string>Choose your file</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTextEdit" name="key_word_input">
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="key_word_btn">
       <property name="text">
        <string>Key word</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>545</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
